evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -1647,22 +1651,19 @@ def test_categories_aggregated_when_some_are_dropped(
 )
 def test_explain_predictions_stacked_ensemble(
     problem_type,
-    dummy_stacked_ensemble_binary_estimator,
-    dummy_stacked_ensemble_multiclass_estimator,
-    dummy_stacked_ensemble_regressor_estimator,
     X_y_binary,
     X_y_multi,
     X_y_regression,
 ):
     if is_binary(problem_type):
         X, y = X_y_binary
-        pipeline = dummy_stacked_ensemble_binary_estimator
+        pipeline = StackedEnsembleClassifier(random_seed=0)
```

Comment:
Also pre-existing but i'm confused: it's called pipeline but is an estimator? Does this code work if its a pipeline with an ensemble estimator? Is that what we're trying to test right now?